tests/playwright/sanity/includes/widgets/rating/helper.ts

```diff
@@ -0,0 +1,27 @@
+import WpAdminPage from '../../../../pages/wp-admin-page';
+
+export async function beforeAll( browser, testInfo, iconExperimentState = 'active' ) {
+	const page = await browser.newPage();
+	const wpAdmin = await new WpAdminPage( page, testInfo );
+
+	await wpAdmin.setExperiments( {
+		container: 'active',
+		rating: 'active',
+		e_font_icon_svg: 'active' === iconExperimentState,
```

Comment:
This comparison is redundant.
```suggestion
		e_font_icon_svg: iconExperimentState,
```

Reply:
@davseve For some reason that doesn't work. I am not sure why. It fails both locally and on the CI.